lib/github/merge/hooks.ex

```diff
@@ -0,0 +1,51 @@
+defmodule BorsNG.GitHub.Merge.Hooks do
+  @moduledoc """
+  Runs shell scripts during Bors's merge process.
+
+  Only invoked when local merges are enabled.
+  """
+
+  defp config do
+    Confex.fetch_env!(:bors, __MODULE__)
+  end
+
+  defp hooks_dir do
+    config()[:hooks_dir]
+  end
+
+  @spec invoke_before_merge_hook!(binary) :: nil
+  def invoke_before_merge_hook!(project_dir) do
+    hook_file =
+      File.cwd!()
+      |> Path.join(project_dir)
+      |> Path.join(hooks_dir())
```

Comment:
What happens if `config()[:hooks_dir]` isn't set?

Reply:
it defaults to `.bors-hooks/`. you can see the default value in `config.exs`.